uint16 DOF
float64[] values